### PR TITLE
Add Sepolia verifier contracts and gateway deployment details

### PR DESCRIPTION
**Pull request by makoto**
NOTE: ENS related contract deployment detail will be added as two separate PRs  (l2 resolvers and reverse registrars will be under `@ensdomains/ens-contracts` and l1 resolvers will be on this repo)

### CODE REVIEW

File: op-verifier/deploy_l1/10_testl1.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   const OPVerifier = await deployments.get('OPVerifier');
   const TestL2 = await hre.companionNetworks['l2'].deployments.get('TestL2');
-
+  console.log([OPVerifier.address, TestL2.address])
```
Comment (Arachnid) [inline op-verifier/deploy_l1/10_testl1.ts:13]: Can you remove this, or if it's needed, change it to include text explaining what the addresses are?

File: arb-gateway/wrangler.toml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name = "arb-gateway-worker"
+name = "arb-sepolia-gateway-worker"
```
Comment (Arachnid) [inline arb-gateway/wrangler.toml:1]: Should this be changed?

**Review by Arachnid — CHANGES_REQUESTED** [review]

File: arb-gateway/wrangler.toml (hunk 2 above)
Comment (makoto) [reply arb-gateway/wrangler.toml:1]: yes. I changed to add testnet info so that `arb-gateway` is used for mainnet

Comment (Arachnid) [reply arb-gateway/wrangler.toml:1]: Right, but this is not a sepolia-specific file; what will we do when deploying to mainnet?

Comment (makoto) [reply arb-gateway/wrangler.toml]: Turned out that I can add environment https://developers.cloudflare.com/workers/wrangler/configuration/#environmental-variables so changed accordingly

**Review by Arachnid — APPROVED** [review]

**Pull request merged**